frontend/src/metabase/query_builder/components/template_tags/TagEditorParamParts/FieldMappingSelect.tsx

```diff
@@ -42,23 +41,19 @@ export function FieldMappingSelect({
 
       {(!hasSelectedDimensionField ||
         (hasSelectedDimensionField && fieldMetadataLoaded)) && (
-        <Schemas.Loader id={table?.schema?.id}>
-          {() => (
-            <SchemaTableAndFieldDataSelector
-              databases={databases}
-              selectedDatabase={database || null}
-              selectedDatabaseId={database?.id || null}
-              selectedTable={table || null}
-              selectedTableId={table?.id || null}
-              selectedField={field || null}
-              selectedFieldId={
-                hasSelectedDimensionField ? tag?.dimension?.[1] : null
-              }
-              setFieldFn={setFieldFn}
-              isInitiallyOpen={!tag.dimension}
-            />
-          )}
-        </Schemas.Loader>
+        <SchemaTableAndFieldDataSelector
```

Comment:
`Schema.Loader` was [added here](https://github.com/metabase/metabase/commit/83f4097e87a9a4ff2b2ce36848daacfe4ec96e07?diff=split&w=1) to fix #19451 but it seemed no longer needed as the repro passes without it now.